.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
                   at: /go/src/graphelier
             - run:
                   name: Build service executable
-                  command: go get graphelier/core/graphelier-service
+                  command: go get -t graphelier/core/graphelier-service/...
```

Comment:
what does this line do ?

Reply:
This builds and installs every package under graphelier-service, along with its regular and test dependencies

Reply:
i guess my question is more for the `/...` at the end of the path

Reply:
Recursively walks subdirectories